config/app.php

```diff
@@ -27,10 +27,14 @@
         ],
         'language-redirect' => [
             'class' => \modules\languageRedirect\LanguageRedirect::class,
+		],
+        'twig-extension' => [
+            'class' => \modules\twigextensionmodule\TwigExtensionModule::class,
```

Comment:
rename `twigextensionmodule` to twigextension please

Reply:
also `TwigExtensionModule` to `TwigExtension`

Reply:
don't forget to change all instences of them too !

Reply:
Won't it clash with the actual TwigExtension class?

Reply:
In therory it should not bc of the "use" statatments at the beginning of each php files.

Reply:
you can have multiple classes named the same way in php. it's kinda weird, but it works with the help of "use" https://www.php.net/manual/en/language.namespaces.importing.php